#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_projectx.dpatch by Tobias Grimm <tg@e-tobi.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Set paths in vdrburn-dvd.sh to make it work with projectx Debian package.

@DPATCH@
diff -urNad vdr-plugin-burn-0.0.009+pre21~/vdrburn-dvd.sh vdr-plugin-burn-0.0.009+pre21/vdrburn-dvd.sh
--- vdr-plugin-burn-0.0.009+pre21~/vdrburn-dvd.sh	2006-09-23 21:23:06.000000000 +0200
+++ vdr-plugin-burn-0.0.009+pre21/vdrburn-dvd.sh	2006-09-23 21:23:45.000000000 +0200
@@ -28,14 +28,6 @@
 # $TEMP_PATH                Full path to burn's temp directory (namely the same
 #                           as $MPEG_TMP_PATH)
 
-if [ -z $JAVA_HOME ]; then
-	export JAVA_HOME=/opt/j2re1.4.2
-fi
-
-if [ -z $PROJECTX_HOME ]; then
-	export PROJECTX_HOME=/opt/ProjectX
-fi
-
 # Some versions of growisofs refuse to start when run with sudo, and may
 # misdetect this condition if vdr itself is started from a sudo session.
 unset SUDO_COMMAND
@@ -75,8 +67,7 @@
 			CUT="-cut $MPEG_DATA_PATH/px.cut"
 		fi
 
-		$JAVA_HOME/bin/java -Djava.awt.headless=true \
-				-jar $PROJECTX_HOME/ProjectX.jar \
+		/usr/bin/projectx \
 				-ini $CONFIG_PATH/ProjectX.ini \
 				$CUT -id $USED_TRACKS,0x1f,0x20 \
 				-demux -out "$MPEG_DATA_PATH" -name vdrsync \
